cli/ballerina-cli-utils/src/main/java/org/ballerinalang/cli/utils/BVMEmbeddedExecutor.java

```diff
@@ -36,7 +38,8 @@
 @JavaSPIService("org.ballerinalang.spi.EmbeddedExecutor")
 public class BVMEmbeddedExecutor implements EmbeddedExecutor {
     @Override
-    public void execute(String programArg, boolean isFunction, String... args) {
+    public long execute(String programArg, boolean isFunction, String... args) {
+        
```

Comment:
Can we remove this extra line?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/10589/commits/d6b6261e3d1cbbe227d0ea1ba7cd2e528cac26d5